py/pykms/pykmsbase.cpp

```diff
@@ -77,10 +77,10 @@ void init_pykmsbase(py::module &m)
 			.def("__repr__", [](const Plane& o) { return "<pykms.Plane " + to_string(o.id()) + ">"; })
 			;
 
-	py::enum_<PlaneType>(m, "PlaneType")
-			.value("Overlay", PlaneType::Overlay)
-			.value("Primary", PlaneType::Primary)
-			.value("Cursor", PlaneType::Cursor)
+	py::enum_<Plane::Type>(m, "Type")
```

Comment:
Doesn't this create a python enum named "Type"? It probably should be "PlaneType".